clojars.el

```diff
@@ -1,33 +1,60 @@
+;;; clojars.el --- clojars.org search interface
+
+;; Filename: clojars.el
+;; Description: clojars.org search interface
+;; Author: Joshua Miller <josh@joshmiller.io>
+;; License: GPLv3
+;; Created: 2014-12-15 17:21:05
+;; Version: x.y.z
```

Comment:
This will need to be changed once the PR is accepted, before the package is submitted to MELPA.